audio/freac/Portfile

```diff
@@ -62,3 +49,23 @@ post-destroot {
     ln -s ${prefix}/bin/smooth-translator \
     ${destroot}${applications_dir}/freac.app/Contents/Resources/translator.app/Contents/MacOS/translator
 }
+
+notes "To reduce the memory footprint, not all optional codecs and tools are installed by default.\
```

Comment:
the term "memory footprint" is strange.... also, how much disk space are we talking about here?

Why did you decide to only keep the `lame` dependency and remove all others? I would think keeping everything enabled by default (i.e., setting this as a default variant) and allowing people who do NOT want to install everything to remove the variant makes more sense.